### ISSUE TITLE
Set default settings // open last used settings

### ISSUE THREAD

**Issue by AlpacaManAlpha** [JarodMica/audiobook_maker#109]
I it possible to change the default TTS Engine?
It would be very convenient.

**Comment by JarodMica** [comment]
A last settings configuration is desired - that's functionality I'm looking to add in a future release

**Comment by AlpacaManAlpha** [comment]
That be a pretty nice quality of life update

**Comment by AlpacaManAlpha** [comment]
@JarodMica while you'r at it, I've noticed, that the engine resets to Tortoise whenever I select a Text file.

**Comment by arrigato1** [comment]
Yes I agree with this, this causes most frustration so far for me, as I open it and run something again and forget it didn't save my last settings.  I'm used to most engines saving your last settings.  So I end up just shutting it down and restarting to get out of it starting on wrong settings.
